refreshVersions/src/main/kotlin/org/gradle/kotlin/dsl/bootstrap.kt

```diff
@@ -0,0 +1,20 @@
+package org.gradle.kotlin.dsl
```

Comment:
that was a good tip from Nikky: no import is necessary when you put code in this package

Reply:
Did you try putting it in the root package?

Reply:
@LouisCAD not necessary, things under `org.gradle.kotlin.dsl` are auto-imported in a `build.gradle.kts` file

Actually we might want to put the dependencies constants under the same package inside the `dependencies` plugin

Reply:
I really don't like that hack as it can be confusing regarding the source. So long as there's no way to add to the implicit imports list, I'll pick root package anytime as it doesn't give false information about the source of the symbol, and encourages investigators to search further instead of being misled.